test/RoadRegistry.BackOffice.Api.Tests/RoadRegistry.BackOffice.Api.Tests.csproj

```diff
@@ -1,7 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
```

Comment:
We don't use the Test.props here which also includes targetframework, but i'm not sure how needed this is.